committee-steering/governance/README.md

```diff
@@ -19,6 +19,23 @@ Specific attention has been given to:
 - Process for agreeing upon technical decisions
 - Process for ensuring technical assets remain healthy
 
+## SIG Charter approval process
+
+When introducing a SIG charter or modification of a charter the following process should be used:
+
+- Identify a small set of owners from the SIG to drive the changes.
+  Most typically this will be the SIG chairs.
```

Comment:
What is the process for a new SIG?  Presumably it folks proposing the SIG agree upon the chairs / tech leads?

Reply:
Unfortunately, this process is split between this doc and `/sig-governance.md`.  We should probably rationalize these but that is a bigger change.  Advice would be appreciated.

I *did* change that in this PR too to say that for creating a SIG a set of folks proposing the SIG should work with the Steering Committee to scope it.